lte/cloud/helm/lte-orc8r/values.yaml

```diff
@@ -142,3 +142,12 @@ subscriberdb_cache:
   service:
     labels: {}
     annotations: {}
+
+nprobe:
```

Comment:
Replicate these values to https://github.com/magma/magma/blob/20eb7d7fa5bd136d9af5f27df3455e22e92d9fd0/lte/cloud/configs/service_registry.yml so they'll work in local dev (Docker) setup